packages/cli/src/__tests__/commands/__snapshots__/Version.test.ts.snap

```diff
@@ -55,3 +55,39 @@ Schema Wasm           : @prisma/prisma-schema-wasm CLI_VERSION.ENGINE_VERSION
 Default Engines Hash  : ENGINE_VERSION
 Studio                : STUDIO_VERSION"
 `;
+
+exports[`version version with driver adapters, it does not show any adapter 1`] = `
+"prisma                : 0.0.0
+@prisma/client        : 0.0.0
+Computed binaryTarget : TEST_PLATFORM
+Operating System      : OS
+Architecture          : ARCHITECTURE
+Node.js               : NODEJS_VERSION
+Query Engine (Binary) : query-engine ENGINE_VERSION (at sanitized_path/query-engine-TEST_PLATFORM, resolved by PRISMA_QUERY_ENGINE_BINARY)
+Schema Engine         : schema-engine-cli ENGINE_VERSION (at sanitized_path/schema-engine-TEST_PLATFORM, resolved by PRISMA_SCHEMA_ENGINE_BINARY)
+Schema Wasm           : @prisma/prisma-schema-wasm CLI_VERSION.ENGINE_VERSION
+Default Engines Hash  : ENGINE_VERSION
+Studio                : STUDIO_VERSION"
+`;
+
+exports[`version version with driver adapters, it only shows official adapters and their underlying drivers 1`] = `
+"prisma                      : 0.0.0
+@prisma/client              : 0.0.0
+@prisma/adapter-d1          : file:../adapter-d1
+ ↳ wrangler                 : 3.5.9
+@prisma/adapter-neon        : file:../adapter-neon
+ ↳ @neondatabase/serverless : 0.9.3
```

Comment:
These versions will not be stable?

Reply:
Those are just example versions from the `package.json` file in the test fixture added by this PR. I can rename them to `<WRANGLER_VERSION>` and `<NEON_VERSION>`, for added clarity.

Reply:
I've updated the tests and description.